### DISCUSSION TITLE
(Wolf CGS 2) Keine ID und keine Zuordnung zur Konfigurationsdatei

### DISCUSSION THREAD

**Discussion by StenmannsAr** [john30/ebusd#1415] [Q&A]
Hallo Zusammen,

ich habe mir den bus adapter shield C6 gekauft um meine Wolf Therme CGS 2 auszulesen.
Sie scheint aber keine ID auszugeben und findet keine passende  Konfiguration.
Die Konfigurations-dateien (csv) habe ich von https://github.com/eBUS/ebus.github.io genommen.

`2024-12-15 11:11:18.461 [bus notice] scan 50: ;Wolf; !;6001;0000
2024-12-15 11:11:18.462 [main error] unable to load scan config 50: no file from wolf with prefix 50 matches ID "", SW6001, HW0000
2024-12-15 11:11:18.462 [main error] scan config 50: ERR: element not found`

Hat jemand einen Tip wie ich die Konfiguration laden kann?

Danke

**Comment by J11522** [comment]
Ich habe das gleiche Problem mit einer Wolf CHA 07.
Diese meldet keine oder eine unvollständige ID. Über einen TIpp wäre ich sehr dankbar.
Anbei ein Austzug von meinem System
```version: ebusd 24.1.p20241110
device: 192.168.178.125:9999, TCP, enhanced, firmware 1.1[4c0f].1[4c0f]
signal: acquired
symbol rate: 41
max symbol rate: 70
min arbitration micros: 4
max arbitration micros: 9
min symbol latency: 4
max symbol latency: 8
scan: finished
reconnects: 0
masters: 4
messages: 21
conditional: 0
poll: 0
update: 10
address 03: master #11
address 08: slave #11, scanned "MF=Wolf;ID= );SW=0160;HW=0000"
address 30: master #3
address 31: master #8, ebusd
address 35: slave #3, scanned "MF=Wolf;ID=;SW=0204;HW=0100"
address 36: slave #8, ebusd
address 50: slave, scanned "MF=Wolf;ID= );SW=0160;HW=0000"
address f1: master #10```

**Reply by Tallman0815** [reply]
Ich habe mit meine Wolf-Heizung das gleiche Problem, allerdings meldet sie sich als Kromschroeder.
Im ebus.wiki steht, wenn keine ID erkannt wird, die Datei ohne genommen wird. Das wäre dann so z.B. 08.csv , geht aber auch nicht.

`address 0c: slave #16, scanned "MF=Kromschroeder;ID=  ;SW=0204;HW=-"
address 10: master #2
address 15: slave #2, scanned "MF=Kromschroeder;ID=  ;SW=0204;HW=-"
address 30: master #3
address 35: slave #3, scanned "MF=Kromschroeder;ID=  ;SW=0204;HW=-"
address 51: slave, scanned "MF=Kromschroeder;ID=  ;SW=0208;HW=-"
address 70: master #4
address 75: slave #4, scanned "MF=Kromschroeder;ID=  ;SW=0204;HW=-"
address f1: master #10
address f6: slave #10, scanned "MF=Kromschroeder;ID=  ;SW=0204;HW=-"
address ff: master #25, ebusd

`

**Reply by J11522** [reply]
Ich konnte inzwischen ohne ID auch erfolgreich Nachrichten lesen. Dazu habe ich die Datei von https://github.com/puni2k/ebusd-configuration-wolf genommen.

Es gibt zwar noch einige Nachrichten die nicht bekannt sind, aber schon viele sind es.

**Comment by Tallman0815** [comment]
Geht leider auch nicht, die einzige Datei die geladen wird scheint die broadcast.csv zu sein.

**Reply by JonesGW-MGD** [reply]
This is totally outside my knowledge zone - as I have neither your equipment nor issue - but do you not need to disable the automatic configuration by removing --scanconfig (https://github.com/john30/ebusd/wiki/4.7.-Automatic-configuration) so that instead it scans the files you have in the --configpath as ebusd use to?  (Please accept my apologies in advance if I muddy the waters rather than clarify!)

**Reply by Tallman0815** [reply]
Since my heater reports itself as Kromschoeder, I assume that the automatic configuration is not working. Therefore, I copied the csv files from the Wolf heater into a local folder. I set the path with --configpath. But no file is loaded, not even when I create a Kromschroeder subfolder.

**Reply by leonundjulie** [reply]
Correct, you have to change from Automatik to Manual, by means you have to save the csv locally and define the configuration path accordingly

**Reply by JonesGW-MGD** [reply]
[@Tallman0815] so if you remove —scanconfig it doesn’t load all the files in —configpath?

(before the --scanconfig option, ebusd use to recursively load al the csv files in the folder.  The auto config section in the wiki I linked to describes this enhancement, but it seems to me you want to skip this and revert to the original behaviour?  You should eliminate unnecessary files from the --configpath folder first  if you do this.)

**Reply by Tallman0815** [reply]
I just want the slave files to be loaded alongside the broadcast.csv.
But that doesn't happen automatically either over the Internet or locally.
I thought that if the heating doesn't send an ID/circuit, the CSV file would just be used without it. For example, 08.csv, but that doesn't work.

**Comment by Tallman0815** [comment]
If you write the CSV file with 2 dots, e.g. 08..csv , it will be loaded even if the heater does not send an ID.

**Comment by realResi** [comment]
Had the same situation — Kromschroeder ID empty, scanconfig not working. Solution is --configpath without --scanconfig, files load directly. Worked fine for my CGG-2-18 setup.

Full configuration here if it helps:
https://github.com/realResi/ebusd-wolf-cgg2
